### ISSUE TITLE
#{} interpolation near operators will be simplified in a future version of Sass

### ISSUE THREAD

**Issue by DaveSkender** [uswds/uswds#4421]
"I'm always frustrated when" I see this in my build logs and developer console window when using uswds.js v2.12.2 (latest) with Jekyll build.

```console
INFO [build-jekyll] DEPRECATION WARNING on line 55 of /tmp/work/site_repo/node_modules/uswds/dist/scss/components/_icon-list.scss:
INFO [build-jekyll] #{} interpolation near operators will be simplified in a future version of Sass.
INFO [build-jekyll] To preserve the current behavior, use quotes:
INFO [build-jekyll] unquote("#{$color}-#{$grade}")
INFO [build-jekyll] DEPRECATION WARNING on line 1408 of /tmp/work/site_repo/node_modules/uswds/dist/scss/core/_functions.scss:
INFO [build-jekyll] #{} interpolation near operators will be simplified in a future version of Sass.
INFO [build-jekyll] To preserve the current behavior, use quotes:
INFO [build-jekyll] unquote("#{$color}-#{$grade}")
```

Short term fix: update offending SCSS file(s) to use suggested quoted syntax

**Comment by aduth** [comment]
I believe this should be fixed as of #4380.

**Comment by brunerae** [comment]
Need to confirm that this issue is ready to be closed.
